BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -134,19 +134,23 @@ extension ElementContent.Builder: ContentStorage {
 
         var result: [(identifier: ElementIdentifier, node: LayoutResultNode)] = []
         result.reserveCapacity(children.count)
+        
+        let identifierFactory = ElementIdentifier.Factory()
 
         for index in 0..<children.count {
-            let currentChildLayoutAttributes = childAttributes[childAttributes.startIndex.advanced(by: index)]
```

Comment:
I don't know why this was written this way before.. Changed...

Reply:
@timdonnelly: In case im missing something, any context on this?

Reply:
If `childAttributes` were a slice or something then using `startIndex` would make sense, but `Layout.layout(..)` always returns an array, so looks equivalent to me